docs/developer-docs/defi/dex/working-with-dex.mdx

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
This page can be deleted, it does not have any unique content that isn't included in the overview page.